lib/fizzy/instantiate.hpp

```diff
@@ -135,7 +135,23 @@ struct ImportedFunction
 /// @a imported_functions may be in any order, but must contain functions for all of the imported
 /// function names defined in the module.
 std::vector<ExternalFunction> resolve_imported_functions(
-    const Module& module, std::vector<ImportedFunction> imported_functions);
+    const Module& module, const std::vector<ImportedFunction>& imported_functions);
+
+// Global that should be used by instantiate as import, identified by module and global name.
+struct ImportedGlobal
+{
+    std::string module;
+    std::string name;
+    Value* value = nullptr;
```

Comment:
Hmm, `module`/`name`/`value` does not seem to be used ye?

Reply:
`find_imported_global` puts it into returned `ExternalGlobal`

Reply:
Ah yes I missed the return value.